UberBarber/UberBarber/View/LoginView.xaml.cs

```diff
@@ -25,31 +25,49 @@ public LoginView()
         {
             InitializeComponent();
         }
-        //This functionality allows you to drag the window to any place
         private void Window_MouseDown(object sender, MouseButtonEventArgs e)
         {
+            //This function allows you to drag the window to any place.
+
             if (e.LeftButton == MouseButtonState.Pressed)
                 DragMove();
         }
-        //Minimize the window
         private void btnMinimize_Click(object sender, RoutedEventArgs e)
         {
+            // This method minimizes the window.
+
             WindowState = WindowState.Minimized;
         }
-        //Close the window
         private void btnClose_Click(object sender, RoutedEventArgs e)
         {
+            // This method closes the whole application.
+
             Application.Current.Shutdown();
         }
 
         private void btnLogin_Click(object sender, RoutedEventArgs e)
         {
+            // TODO: correct doctring
             //open new window if given credentials are correct
             DatabaseQueries query = new DatabaseQueries();
 
-            if (!query.logging(txtUser.Text, txtPassword.Password)) return;
-            new MainWindow().Show();
-            this.Close();
+            try
+            {
+                query.Logging(txtUser.Text, txtPassword.Password);
+                new MainWindow().Show();
+                Close();
+            }
+            catch (Exception ex)
```

Comment:
logging function is secured well, only thing here to catch is window opening. Specify exception type here

Reply:
done